jobs/JGDAS_ENKF_ECEN

```diff
@@ -20,35 +20,23 @@ GDATE=$(${NDATE} -${assim_freq} ${CDATE})
 gPDY=$(echo ${GDATE} | cut -c1-8)
 gcyc=$(echo ${GDATE} | cut -c9-10)
 GDUMP=${GDUMP:-"gdas"}
+GDUMP_ENS=${GDUMP_ENS:-"enkf${GDUMP}"}
 
 export CASE=${CASE_ENKF}
 
+export CDUMP_OBS=${CDUMP_OBS:-${CDUMP/enkf}}
```

Comment:
`CDUMP_OBS` is needed for?

Reply:
This was done because this is how other j-jobs did it (though I clearly wasn't consistent, since I didn't use a `CDUMP_CTL` and instead just did the substitution in-line). I didn't worry _too_ much about consistency since I know this is all about to be substantially refactored anyway.